client/state/stats/lists/utils.js

```diff
@@ -275,6 +275,26 @@ function parseChartData( payload, nullAttributes = [] ) {
 	} );
 }
 
+function parseStoreReferrerData( payload ) {
```

Comment:
Could we have some unit tests for this new util? I've written some buggy utils in this file in the past that have lead to exceptions 🤕